app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -247,4 +261,8 @@ class MainFragment : Fragment() {
             }
         }
     }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        StoreSession.readInt(AppConstants.BEGIN_STATUS)
```

Comment:
What is the need for this line here?

Reply:
I used this line to so that when app destroys, begin status retains its value. I tried without it few days back, but it wasn't retaining the value after killing the app.

Reply:
Then on searching I came across that we have to use savedInstance in OnCreate()